shopfloor_mobile/controllers/main.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from odoo import http
+
+
+class ShopfloorMobileAppController(http.Controller):
+
+    module_path = os.path.split(os.path.dirname(os.path.realpath(__file__)))[0]
+
+    @http.route(
+        ["/shopfloormobile/scanner/<path:path_fragment>", "/shopfloormobile/scanner"],
+        auth="public",
+    )
+    def load_app_and_assets(self, path_fragment=""):
```

Comment:
wouldn't be better to authorize via api key?
We should make public only the login. No?

Reply:
yes